contrib/cdisetup/nvidia/nvidia.go

```diff
@@ -94,8 +90,11 @@ func (s *setup) Run(ctx context.Context) (err error) {
 	}
 
 	var needsDriver bool
-
-	if _, err := os.Stat("/proc/driver/nvidia"); err != nil {
+	if nvidiaSmi, err := exec.LookPath("nvidia-smi"); err == nil && nvidiaSmi != "" {
```

Comment:
why are we checking this first? Isn't this supposed to be `nvidiaSmi && isWSL`

Reply:
Now checking if drivers are already installed at expected location. If not then check `nvidia-smi` exec does not fail which means drivers are not needed.